sdk/contentsafety/Azure.AI.ContentSafety/src/Customizations/ContentSafetyImageData.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.AI.ContentSafety
+{
+    public partial class ContentSafetyImageData
+    {
+        internal Uri BlobUri { get; }
```

Comment:
Why customize those to internal which will not be able to access outside the sdk library?
and After the customization, have you regen the sdk? There are the same properties in Generated\ContentSafetyImageData.cs which will cause build fail.